docs/operating-scylla/procedures/cluster-management/scale-up-cluster.rst

```diff
@@ -2,6 +2,12 @@
 Upscale a Cluster
 =================
 
+.. note::
+
+   Upscaling a cluster requires at least a quorum of nodes in a cluster to be available. 
+   If the quorum is lost, it must be restored before a new node is added. 
+   See :doc:`Handling Node Failures </troubleshooting/handling-node-failures>` for details. 
```

Comment:
I would move this note to the "Upscale Nodes by Adding CPUs" section.

The "Add Bigger Nodes to a Cluster" case defers to other articles which already contain appropriate warnings. In fact this case requires ALL nodes to be present, not just quorum.

The "Upscale Nodes by Adding CPUs" section requires quorum.

In fact I believe we should make it more clear in the "Upscale Nodes by Adding CPUs" that the nodes should be restarted **sequentially** because we don't want to interrupt applications (see my other comment about rolling restart).